tests/test_blog.py

```
@@ -0,0 +1,386 @@
+        with patch('services.blog.Path') as mock_path:
+            def mock_path_side_effect(path_str):
+                if path_str == "data":
+                    return temp_data_dir
+                elif path_str == "blogs":
+                    return temp_blogs_dir
+                else:
+                    return Path(path_str)
+            
+            mock_path.side_effect = mock_path_side_effect
+            
+            builder = BlogDigestBuilder()
```

Ruff (0.12.2)

Local variable `builder` is assigned to but never used

Remove assignment to unused variable `builder`

(F841)